apps/backoffice-v2/package.json

```diff
@@ -56,7 +56,7 @@
     "@lukemorales/query-key-factory": "^1.0.3",
     "@radix-ui/react-checkbox": "^1.0.1",
     "@radix-ui/react-dialog": "^1.0.2",
-    "@radix-ui/react-dropdown-menu": "^2.0.4",
+    "@radix-ui/react-dropdown-menu": "^2.0.5",
```

Comment:
Otherwise opening a dialog and closing it results in no pointer events on the whole app and no ability to scroll.